golem-cloud-server-oss/docker/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM debian:bookworm-slim as base
+
+ARG TARGETARCH
+
+FROM base AS platform-amd64
+
+ENV RUST_TARGET=x86_64-unknown-linux-gnu
+
+FROM base AS platform-arm64
+
+ENV RUST_TARGET=aarch64-unknown-linux-gnu
+
+FROM platform-${TARGETARCH} AS final
+
+WORKDIR /app
+COPY /target/$RUST_TARGET/release/cloud-server-oss ./
+COPY /golem-cloud-server-oss/config/cloud-server.toml ./config/cloud-server.toml
+
+RUN apt-get update && apt-get install -y libssl-dev
+RUN apt-get update && apt-get install -y ca-certificates
+RUN update-ca-certificates
+
+EXPOSE 8080
+EXPOSE 9090
+
+ENTRYPOINT ["./cloud-server-oss"]
```

Comment:
I can't quite understand this dockerFile

Reply:
TARGETARCH is provided by buildkit see: https://docs.docker.com/engine/reference/builder/#automatic-platform-args-in-the-global-scope

platforms: linux/amd64,linux/arm64
archs: amd64, arm64

based on `platform-${TARGETARCH}` then there will be `RUST_TARGET=x86_64-unknown-linux-gnu` or `RUST_TARGET=aarch64-unknown-linux-gnu`

which will be used in `COPY /target/$RUST_TARGET/release/cloud-server-oss ./`